src/ipc/prepare-boot-usb.ts

```diff
@@ -1,14 +1,14 @@
 import makeDebug from 'debug'
-import { IpcMain, IpcMainInvokeEvent } from 'electron'
+import { IpcMain } from 'electron'
 import exec from '../utils/exec'
-import { join } from 'path'
 
 export const channel = 'prepare-boot-usb'
 const debug = makeDebug('kiosk-browser:prepare-boot-usb')
 
-const BootableOptionPattern = /^Boot([0-9]+)\*\s(.+)\s(.+)$/
+const BootableOptionPattern = /^Boot([0-9a-fA-F]+)\*?\s(.+)\s(.+)$/
```

Comment:
Why make the * optional?

Reply:
I can't figure out what it means, before I thought it might be an entry which isn't actually bootable as thats how I've seen it on my own testing (like EFI CD Device when there is no CD) but looking at other random peoples output online I'm less sure and in this one https://askubuntu.com/questions/489967/explanation-of-my-efibootmgr-v-output I noticed that there "Boot Menu" didn't have it so then I thought it might be safer to make it optional. 

I also was looking at someone elses regex which included the A-F and made it optional.

Reply:
oh actually just saw in that thread someone saying it means active, so maybe I should keep requiring it? Does your boot menu entry have the * ?

Reply:
`Boot Menu` entry does NOT have a *, so yeah you've got to make it work in that case.

Reply:
Cool I'll update the test to reflect that